### ISSUE TITLE
MoveException: The [to] path already exists. Use overwrite:true 

### ISSUE THREAD

**Issue by whoizit** [noojee/dcli#173]
```
dcli version
Dcli Version: 1.13.1, Located at: /home/user/.pub-cache/bin/dcli
```
I have:
```bash
$ ls ~/bin/*.dart
footq.dart
```
I do:
```bash
~ > rm ~/bin/footq ~/.dcli/bin/footq
~ > cd ~/bin
~/bin > dcli compile --install --overwrite

Compiling /home/user/bin/footq.dart...

path: /home/user/bin/footq.dart
Resolving dependencies...
Got dependencies!
Info: Compiling with sound null safety
Generated: /home/user/bin/footq

Installing /home/user/bin/footq into /home/user/.dcli/bin/footq
```
and then I compile again
```
~/bin > dcli compile --install --overwrite

Compiling /home/user/bin/footq.dart...

path: /home/user/bin/footq.dart
Resolving dependencies...
Got dependencies!
Info: Compiling with sound null safety
Generated: /home/user/bin/footq

Installing /home/user/bin/footq into /home/user/.dcli/bin/footq
MoveException: The [to] path /home/user/.dcli/bin/footq already exists. Use overwrite:true  
Stacktrace: wait_for_ex.dart : waitForEx : 62
move.dart : move : 26
dart_script.dart : DartScript.compile : 292
compile.dart : CompileCommand.compileScript : 130
compile.dart : CompileCommand.run : 58
command_line_runner.dart : CommandLineRunner.process : 86
entry_point.dart : EntryPoint._parseCmdLine : 35
entry_point.dart : EntryPoint.process : 30
dcli.dart : DCli.run : 13
dcli.dart : main : 8
```
this bug appeared several releases ago

**Comment by whoizit** [comment]
seems like --overwrite option not working anymore

**Comment by bsutton** [comment]
Fixed in 1.13.2
